src/components/clusters/ClusterDetailsMultiRegion/components/ClusterDetailsTop/components/RecommendedOperatorsAlert/RecommendedOperatorsAlert.tsx

```diff
@@ -26,6 +31,7 @@ type RecommendedOperatorsAlertProps = {
   closeDrawer: () => void;
   onDismissAlertCallback: () => void;
   consoleURL?: string;
+  cluster: ClusterWithPermissions;
```

Comment:
It's clearer to only pass down what's needed for the analytics purpose instead of passing the whole cluster. Similarly to how it was done for `ErrorBox.tsx` in this PR.